src/components/ElementCard.vue

```diff
@@ -0,0 +1,143 @@
+<script setup lang="ts">
+    import { storeTMDB } from '../stores/storePinia'
+    import colorGenre from '../utils/colorGenre'
+    import {
+        MovieRequest,
+        TypeOfGenre,
+        ColorToEachGenre,
+    } from '../types/apiType'
+    import { onMounted, ref } from 'vue'
+
+    interface Props {
+        allInfosMovie: MovieRequest
+    }
+    const props = defineProps<Props>()
+    const genres = ref()
+    const colorForGenre: ColorToEachGenre = colorGenre
+
+    onMounted(() => {
+        if (props.allInfosMovie && props.allInfosMovie?.genre_ids) {
```

Comment:
Tu peux écrire juste ça
```suggestion
        if (props.allInfosMovie?.genre_ids) {
```